.github/workflows/lint.yml

```diff
@@ -23,12 +23,12 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version: '3.11'
+          python-version: "3.11"
```

Comment:
The change itself is fine, although the commit message is unrelated to the change. It's trivial and not important, but we should try to avoid this in the future by having good commit messages ;-).